pallets/remote-asset-manager/src/mock.rs

```diff
@@ -556,3 +610,224 @@ pub mod para {
         }
     );
 }
+
+pub mod statemint {
```

Comment:
Could we move mock runtimes like this into one crate?

Reply:
that's not trivial because they use the router that gets generated within the `decl_test_network` macro. that's also the reason why import the relay runtime like this.
we could extract it to a separate mod though.

Reply:
Maybe we can write another macro to generate these?

Reply:
not possible currently, because of circular dependencies: parachain runtimes depend on `Router` that gets generated within the `decl_test_network`.
My guess is that `xcm-simulator` will address this, therefor we can hold off on this.